example/timer/odp_timer_accuracy.c

```diff
@@ -231,13 +230,14 @@ static int start_timers(test_global_t *test_global)
 		event = odp_timeout_to_event(timeout);
 
 		if (i == 0) {
-			first_tick = odp_timer_current_tick(timer_pool);
+			start_tick = odp_timer_current_tick(timer_pool);
 			time       = odp_time_local();
-			first_ns   = odp_time_to_ns(time);
-			test_global->first_ns = first_ns + START_OFFSET_NS;
+			start_ns   = odp_time_to_ns(time);
+			test_global->first_ns = start_ns + START_OFFSET_NS;
 		}
 
-		tick = first_tick + offset_tick + (i * period_tick);
+		time_ns = test_global->first_ns + (i * period_ns);
+		tick = start_tick + odp_timer_ns_to_tick(timer_pool, time_ns);
 		ret = odp_timer_set_abs(timer, tick, &event);
 
 		if (ret != ODP_TIMER_SUCCESS) {
```

Comment:
while reviewing this I see that odp_timer_free() is missing. Not related to this issue but this function has returns with allocated timer and timeouts.

Reply:
All destroys are handled in destroy_timers() function